app/src/main/res/values/component_colors.xml

```diff
@@ -36,6 +36,19 @@
   <color name="component_color_admin_pin_activity_layout_background_color">@color/color_palette_background_color</color>
   <!-- Pin Password Activity -->
   <color name="component_color_pin_password_activity_layout_background_color">@color/color_palette_background_color</color>
+  <!-- Third Party Dependency List Activity -->
+  <color name="component_color_third_party_dependency_list_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_third_party_dependency_options_list_activity_background_color">@color/color_palette_container_background_color</color>
+  <color name="component_color_third_party_dependency_items_text_list_activity_background_color">@color/color_palette_dark_text_color</color>
+  <color name="component_color_third_party_dependency_version_text_view_list_activity_color">@color/color_palette_primary_text_color</color>
+  <!-- License List Activity -->
+  <color name="component_color_license_list_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_license_list_activity_items_background_color">@color/color_palette_container_background_color</color>
+  <!-- License List Text Viewer Activity -->
+  <color name="component_color_license_list_text_viewer_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_license_list_text_viewer_items_activity_background_color">@color/color_palette_dark_text_color</color>
+  <color name="component_color_license_text_viewer_list_options_activity_background_color">@color/color_palette_container_background_color</color>
+  <color name="component_color_license_text_viewer_text_color">@color/color_palette_description_text_color</color>
```

Comment:
Naming of all components is sort of incorrect. Follow this format

`component_color_<activity-name>_<type-of-color>_color`

Example

`component_color_third_party_dependency_list_activity_background_color ` -> this is correct
`component_color_third_party_dependency_version_text_view_list_activity_color ` should be renamed to `component_color_third_party_dependency_list_activity_version_text_color `

Basically all internal layouts like `fragments`, `items` will also use `activity` name only.

Example `component_color_license_list_text_viewer_activity_background_color ` -> `component_color_license_text_viewer_activity_background_color `

Reply:
done